GoogleForCreators/web-stories-wp
# Respect first day of the week setting

## Context



## Summary

Respect first day of the week setting for calender. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Go to settings > general. 
2. Change first day of the week. 
3. Go to story editor. 
4. Go to documents panel. 
5. Change date of publish.
6. See first day of the week has changed. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11651
